tests/monitoring/monitoring_service/test_handlers.py

```diff
@@ -95,7 +95,7 @@ def get_signed_monitor_request(
     )
     monitor_request = UnsignedMonitorRequest.from_balance_proof(
         bp,
-        reward_amount=0,
+        reward_amount=10,
```

Comment:
Can you make this a parameter of the function as well?

Reply:
yes, sure we talked about that

Reply:
done